yaml/prometheus.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus
+  namespace: openfaas
+  labels:
+    app: prometheus
+spec:
+  type: NodePort
+  ports:
+    - port: 9090
+      protocol: TCP
+      targetPort: 9090
+      nodePort: 31119
+  selector:
+    app: prometheus
+---
+apiVersion: apps/v1beta1 # for versions before 1.6.0 use extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus
+  namespace: openfaas
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: prometheus
+    spec:
+      containers:
+      - name: prometheus
+        image: functions/prometheus:latest-k8s
```

Comment:
If you look at the PR from @weikinhuang he moves us to the official image too, but I want to focus on one change at a time. If we can prove this out we'll then move forward - I think it would probably make sense to jump straight to 2.0 with adequate testing.